Troubleshooting.md

```diff
@@ -189,4 +191,22 @@ If you want to prevent using reflection on third-party classes in the future you
 
 **Reason:** You used `GsonBuilder.excludeFieldsWithModifiers` to overwrite the default excluded modifiers
 
-**Solution:** When calling `GsonBuilder.excludeFieldsWithModifiers` you overwrite the default excluded modifiers. Therefore, you have to explicitly exclude `static` fields if desired. This can be done by adding `| Modifier.STATIC` to the argument.
+**Solution:** When calling `GsonBuilder.excludeFieldsWithModifiers` you overwrite the default excluded modifiers. Therefore, you have to explicitly exclude `static` fields if desired. This can be done by adding `Modifier.STATIC` as additional argument.

```

Comment:
I have changed this because I forgot to consider that the `excludeFieldsWithModifiers` parameter is varargs so it is easier to just pass `Modifier.STATIC` as separate argument than having to do bitwise operations.